File: pback/schemas/service.py
from typing import Optional

from pydantic import BaseModel as BM


class CreateService(BM):
    name: str
    price: Optional[float]
    price_lower_bound: Optional[float]
    price_higher_bound: Optional[float]
    seconds: int
    description: Optional[str]


class OutService(BM):
    service_id: int
    name: str
    price: Optional[float]
    price_lower_bound: Optional[float]
    price_higher_bound: Optional[float]
    seconds: int
    description: str

    class Config:
        orm_mode = True

File: vfront/src/models/service.ts
export class Service {
    constructor(
        public service_id: number,
        public name: string,
        public price: number,
        public currency: string,
        public seconds: number,
        public price_lower_bound?: number,
        public price_higher_bound?: number,
        public description?: string, 
    ){}
}